cmd/repo-updater/repoupdater/server.go

```diff
@@ -78,6 +79,15 @@ func (s *Server) Handler() http.Handler {
 	return mux
 }
 
+func (s *Server) handleHealthz(w http.ResponseWriter, r *http.Request) {
+	w.WriteHeader(200)
+	_, err := w.Write([]byte("ok"))
+	if err != nil {
+		log15.Info("Error checking /healthz: " + err.Error())
```

Comment:
```suggestion
		log15.Error("Error checking /healthz", "error", err)
```

Unless it's intentional.

Reply:
If so, please add a code comment so people come after won't be confused :)

Reply:
No, I copy/pasted this from one of our other services to keep it consistent.

Reply:
I would advocate for switching it over to structured, but maybe keep it as a warning